app/manifest.json

```diff
@@ -18,7 +18,8 @@
     "storage",
     "*://*.bungie.net/*",
```

Comment:
If you're changing the permissions, you should update this to be *://www.bungie.net/*